python/example_code/lookoutvision/inference.py

```diff
@@ -95,50 +83,204 @@ def download_from_s3(s3_resource, photo):
 
         return local_file
 
+    @staticmethod
+    def reject_on_classification(image, prediction, confidence_limit):
+        """
+        Returns True if the anomaly confidence is greater than or equal to 
+        the supplied confidence limit.
+        :param image: The name of the image file that was analyzed.
+        :param prediction: The DetectAnomalyResult object returned from DetectAnomalies
+        :param confidence_limit: The minimum acceptable confidence. Float value between 0 and 1.
+        :return: True if the error condition indicates an anomaly, otherwise False.
+        """
+
+        reject = False
+
+        logger.info("Checking classification for %s", image)
+
+        if prediction['IsAnomalous'] and prediction['Confidence'] >= confidence_limit:
+            reject = True
+            reject_info=(f"Rejected: Anomaly confidence ({prediction['Confidence']:.2%}) is greater"
+                    f" than limit ({confidence_limit:.2%})")
+            logger.info("%s", reject_info)
+
+        if not reject:
+            logger.info("No anomalies found.")
+        return reject
+
+    @staticmethod
+    def reject_on_anomaly_types(image, prediction, confidence_limit, anomaly_types_limit):
+        """
+        Checks if the number of anomaly types is greater than than the anomaly types
+        limit and if the prediction confidence is greater than the confidence limit.
+        :param image: The name of the image file that was analyzed.
+        :param prediction: The DetectAnomalyResult object returned from DetectAnomalies
+        :param confidence: The minimum acceptable confidence. Float value between 0 and 1.
```

Comment:
Should "Float value between 0 and 1." be changed to “(Float 0-1).” to match above?